index.js

```diff
@@ -53,7 +55,7 @@ function gulpFlatBlog(templates) {
         post.slug = file.relative.substring(0, file.relative.lastIndexOf('.'));
       }
 
-      file.contents = new Buffer(templates.single(post));
+      file.contents = new Buffer(templates.single({post: post, index: pageIndex}));
```

Comment:
I think some users may find using `index` as the name confusing in their templates, as several templating engines use `index` as some kind of keyword (e.g. `@index` in handlebars). May I suggest `post` and `posts`?